recipe/conda_build_config.yaml

```diff
@@ -1,3 +1,6 @@
 MACOSX_DEPLOYMENT_TARGET:  # [osx]
 - '11.0'                   # [osx and arm64]
-- '10.13'                  # [osx and x86_64]
\ No newline at end of file
+- '10.13'                  # [osx and x86_64]
+occt:
+  - 7.7.2
+  - 7.6.3
```

Comment:
what about just unpinning?

Reply:
we wanted to build against 7.6.3 and 7.7.2. There are some problems reported with 7.7.2 that's why we still want to keep 7.6.3